packages/@textlint/markdown-to-ast/test/fixtures/bom.text/output.json

```diff
@@ -11,35 +11,35 @@
                     "loc": {
                         "start": {
                             "line": 1,
-                            "column": 3
+                            "column": 2
                         },
                         "end": {
                             "line": 1,
-                            "column": 19
+                            "column": 18
                         }
                     },
                     "range": [
-                        3,
-                        19
+                        2,
+                        18
                     ],
-                    "raw": "Hello from a BOM"
+                    "raw": " Hello from a BO"
```

Comment:
- [ ] This is wrong probably, slicing text with range

Reply:
https://github.com/textlint/textlint/pull/767/commits/68fbae210471b18f9861285c5408d4e062244bcf fix it

Reply:
Fmm, https://github.com/remarkjs/remark/blob/25c7a324403afc68cdf54ab85d566516228cc639/test/fixtures/tree/bom.json#L12-L21 remark does not consider the offset.

Reply:
https://github.com/textlint/textlint/pull/767/commits/09630c90685848648fd59fdd7e9936ff48190513 fix it